deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -32,9 +32,10 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args: ["ceph", "operator"]
         securityContext:
-          runAsNonRoot: true
-          runAsUser: 2016
```

Comment:
The operator image has user 2016 added [here](https://github.com/rook/rook/blob/master/images/ceph/Dockerfile#L34). Why remove this user?

Reply:
It's just convention. See https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

Using a different user, group and fs group should be better for security.